Services/Mail/classes/class.ilFormatMail.php

```diff
@@ -95,7 +95,7 @@ public function formatReplyRecipientsForCC()
     * @access	public
     * @return string
     */
-    public function formatReplyRecipient()
+    public function formatReplyRecipient() : mixed
```

Comment:
The `mixed` type is not supported in PHP 7.4, which we'll have to support as well in ILIAS 8.